huggingface-transformers/requirements.txt

```diff
@@ -0,0 +1,4 @@
+transformers==4.41.1
+torch==2.2.0
+accelerate==0.30.1
+polars==0.20.31
```

Comment:
Polars just moved to v1.

If you have the time, it would be nice to switch to the [latest Polars](https://pypi.org/project/polars/) and just make sure everything still works as expected. If we pull in v1.1.0, that includes support for NumPy v2 as well, which adds another "challenge" it would be nice to see if we pass.

Reply:
I upgraded to polars v1.1.0 and everything seems to work!